README_BCtable.md

```diff
@@ -96,10 +104,16 @@ output stream to be used by the next filter. This is a list of available filters
 **convert2grb**
 :  Converts the stream to GRIB format
 
-**day_to_6h**
-:  Temporal interpolation in time from daily to 6-hourly frequency. Recommended
+**Mh_to_Nh**
+:  Interpolation in time from M-hourly frequency to N-hourly frequency (M > N).
+
+**day_to_Nh**
```

Comment:
Given that you defined the `interval` variable in the BCtable, we could call this filter `day_to_interval`, and the same for other `Nh` filters

Reply:
Yes. I am testing at the moment if these 2 filters (Mh_to_Nh and  day_to_Nh) can be merged into one. If so, the name "time_interp_to_interval" will be set as the name of the filter.